litellm/proxy/_types.py

```diff
@@ -77,6 +77,16 @@ class LiteLLM_UpperboundKeyGenerateParams(LiteLLMBase):
 
 
 class LiteLLMRoutes(enum.Enum):
+    openai_route_names: List = [
+        "chat_completion",
+        "completion",
+        "embeddings",
+        "image_generation",
+        "audio_transcriptions",
+        "moderations",
+        "model_info_v1",
```

Comment:
do you mean the `/v1/models` route - which is the openai-compatible endpoint 

or the litellm-specific `/model/info` route which is used for model management? 

given the name here, i'd assume you mean the `/v1/models` route? @msabramo

Reply:
Thanks! Yes, I got a little confused. Updated the routes in b079f4cb7.